modules/reindex/src/test/java/org/opensearch/index/reindex/RestReindexActionTests.java

```diff
@@ -74,7 +74,7 @@ public void testPipelineQueryParameterIsError() throws IOException {
                 body.endObject();
             }
             body.endObject();
-            request.withContent(BytesReference.bytes(body), body.contentType());
+            request.withContent(BytesReference.bytes(body), (XContentType) body.contentType());
```

Comment:
It seems like `withContent` could be changed to accept `MediaType`, it does not need `XContentType` at all